src/crewai/agents/executor.py

```diff
@@ -305,3 +314,32 @@ def _ask_human_input(self, final_answer: dict) -> str:
         return input(
             self._i18n.slice("getting_input").format(final_answer=final_answer)
         )
+
+    def _training_handler(
+        self, output: AgentFinish, human_feedback: str | None = None
+    ) -> None:
+        """Function to handle the process of the training data."""
+        training_data_filename = "training_data.pkl"
```

Comment:
Should we have these `pkl` file names in consts, maybe? This way we have a single place to find them.

Reply:
Definitely, I'll add it. Thanks @gvieira